specs/phase0/p2p-interface.md

```diff
@@ -561,10 +560,9 @@ The request MUST be encoded according to the encoding strategy.
 The requester MUST close the write side of the stream once it finishes writing the request message.
 At this point, the stream will be half-closed.
 
-The requester MUST wait a maximum of `TTFB_TIMEOUT` for the first response byte to arrive (time to first byte—or TTFB—timeout).
-On that happening, the requester allows a further `RESP_TIMEOUT` for each subsequent `response_chunk` received.
+The requester MUST NOT make more than two concurrent requests with the same ID.
```

Comment:
How about making the number two a config variable?

Reply:
+1
Embedded in the spec it looks like a fundamental constant which is not subject to change (kind a number of children in a binary tree 😄 )

Reply:
I'm not really a fan of pre-emptive configurability (one could add this at any time when it's deemed useful) but there you go: [911019c](https://github.com/ethereum/consensus-specs/pull/3767/commits/911019c41f35aee666b07a558a46bbc033951f20)